README.rst

```diff
@@ -4,8 +4,8 @@
 
 Toil is a scalable, efficient, cross-platform (Linux & macOS) pipeline management system,
 written entirely in Python, and designed around the principles of functional
-programming. It supports running workflows written in either Common Workflow Language (`CWL`_) or 
-Workflow Description Language (`WDL`_), as well as having its own rich Python API for writing workflows against. 
+programming. It supports running workflows written in either Common Workflow Language (`CWL`_) 1.0-1.2 or 
+Workflow Description Language (`WDL`_) 1.0-1.1, as well as having its own rich Python API for writing workflows against. 
```

Comment:
WDL or OpenWDL? They are different things..

Reply:
My understanding is that "OpenWDL" is an organization, and that "WDL" is a language, with "WDL 1.0" and "WDL 1.1" being language specifications published by OpenWDL.

Is there another "WDL 1.0" that isn't the same as the one OpenWDL wrote a spec for?

Reply:
My understanding is that WDL draft 2, WDL 1.0, and "WDL development" is what Cromwell implements. OpenWDL 1.0 is different from (Cromwell) WDL 1.0. Cromwell doesn't support OpenWDL 1.1.

Reply:
On [Cromwell's page about language support](https://cromwell.readthedocs.io/en/latest/LanguageSupport/) they link to OpenWDL's spec for Draft 2. Or at least where it would be if it was still where they link to.

So I don't think Cromwell's WDL version number series is genuinely intended to be a different language or set of language versions.

Now, "WDL 1.0 as implemented by Cromwell" is not 100% identical to "WDL 1.0 as specified by OpenWDL". But I don't think that that means Cromwell thinks they have implemented the "real" WDL 1.0, it just means the spec and one of the popular implementations don't 100% match. And I haven't seen anyone on the OpenWDL side calling the *language* OpenWDL.